packages/react-native/cli.js

```diff
@@ -39,6 +39,16 @@ async function getLatestVersion(registryHost = DEFAULT_REGISTRY_HOST) {
   });
 }
 
+/**
+ * Warn when users are using `npx react-native init`, to raise awareness of the changes from RFC 0759.
+ * @see https://github.com/react-native-community/discussions-and-proposals/tree/main/proposals/0759-react-native-frameworks.md
```

Comment:
Note, this is pointing to the merged path, which is currently unavailable. The RFC is scheduled to be merged on the [26th of Feb](https://github.com/react-native-community/discussions-and-proposals/pull/759#issuecomment-1952932177).